File: VB/DXApplication1/Reports/ReportPreview.ascx.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.UI
Imports System.Web.UI.WebControls

Namespace DXApplication1.Reports
	#Region "#reportpreview"
	Partial Public Class ReportPreview
		Inherits System.Web.UI.UserControl
		Private privateControlAdapter As DevExpress.Web.ASPxScheduler.Reporting.ASPxSchedulerControlPrintAdapter
		Public Property ControlAdapter() As DevExpress.Web.ASPxScheduler.Reporting.ASPxSchedulerControlPrintAdapter
			Get
				Return privateControlAdapter
			End Get
			Set(ByVal value As DevExpress.Web.ASPxScheduler.Reporting.ASPxSchedulerControlPrintAdapter)
				privateControlAdapter = value
			End Set
		End Property

		Protected Sub Page_Init(ByVal sender As Object, ByVal e As EventArgs)
			Dim rpt As New XtraSchedulerReport1()
			rpt.SchedulerAdapter = ControlAdapter.SchedulerAdapter
			ASPxDocumentViewer1.Report = rpt
		End Sub
	End Class
	#End Region ' #reportpreview
End Namespace

File: CS/DXApplication1/Reports/ReportPreview.ascx.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;

namespace DXApplication1.Reports
{
    #region #reportpreview
    public partial class ReportPreview : System.Web.UI.UserControl
    {
        public DevExpress.Web.ASPxScheduler.Reporting.ASPxSchedulerControlPrintAdapter ControlAdapter { get; set; }

        protected void Page_Init(object sender, EventArgs e)
        {
            XtraSchedulerReport1 rpt = new XtraSchedulerReport1();
            rpt.SchedulerAdapter = ControlAdapter.SchedulerAdapter;
            ASPxDocumentViewer1.Report = rpt;
        }
    }
    #endregion #reportpreview
}